### ISSUE TITLE
IE10

### ISSUE THREAD

**Issue by LGprodigy** [clappr/clappr#560]
I'm experiencing a bit of a funny glitch. Really hard to explain... I am using HLS streams and it seems to be okay... except when I go to play the video. I click the poster and nothing happens... then I double click and the poster goes away... and nothing happens. It works just fine in IE11

**Comment by flavioribeiro** [comment]
@LGprodigy do u have a valid URL where we can reproduce this behavior?

**Comment by michaelcunningham19** [comment]
@flavioribeiro 
@LGprodigy 

Same issue as here? https://github.com/clappr/clappr/issues/788

**Comment by philcrump** [comment]
This also looks like the same bug I mentioned as part of #637

**Comment by philcrump** [comment]
And therefore reproducible on http://cdn.clappr.io/ in IE10 using only the change:
```
  source: 'http://www.streambox.fr/playlists/x36xhzz/x36xhzz.m3u8',
```

**Comment by philcrump** [comment]
As of the 0.2.34 the bug I referred to is fixed. Thanks @michaelcunningham19 for his fixes in #788!
